Pod/Classes/WPNavigationMediaPickerViewController.m

```diff
@@ -253,7 +253,7 @@ - (void)updateSelectionAction {
 - (NSString *)selectionActionValue {
     NSString *actionString = self.selectionActionTitle;
     if (actionString == nil) {
-        actionString = NSLocalizedString(@"Select %@", @"");
+        actionString = NSLocalizedString(@"Select %@", @"Action for Navigation bar to indicate selection of media. The argument in the string is a numeral number that indicates the number of elements selected.");
```

Comment:
You can probably shorten the last bit of this to "The argument in the string represents the number of elements selected."

Reply:
I wanted to say it's digits like this: 1 and not like this: one, two.

Reply:
Ah. Perhaps we can say "The argument in the string represents the number of elements (as numeric digits) selected."